src/containers/common/navigation.js

```diff
@@ -1,47 +1,56 @@
 import React from 'react';
 import { Link } from "react-router-dom";
 import StaffBookLogo from '../../assets/staff-book-logo.png';
+import CandidateStatusIcon from '../../assets/icons/candidate-status.png';
+import DashboardIcon from '../../assets/icons/dashboard.png';
+import NotificationIcon from '../../assets/icons/notification.png';
+import RecommendedProfileIcon from '../../assets/icons/recommended-profile.png';
+import SearchProfileIcon from '../../assets/icons/search-profile.png';
+import SettingsIcon from '../../assets/icons/settings.png';
+import ShortlistedProfileIcon from '../../assets/icons/shortlisted-profile.png';
 
 const Navigation = (props) => {
   return (
     <nav className="navbar navbar-expand-lg">
-      <a className="navbar-brand" href="/">
+      <Link className="navbar-brand" to="/">
         <img src={StaffBookLogo} height="80" alt="" />
-      </a>
+      </Link>
       <div className="collapse navbar-collapse" id="navbarNav">
         <ul className="navbar-nav ml-auto">
           <li className="nav-item active">
-            <a className="nav-link" href="#"><i className="fa fa-th-large" /> Dashboard</a>
+            <Link className="nav-link" to="/">
+              <img src={DashboardIcon} height="15px"/> Dashboard
+            </Link>
           </li>
           <li className="nav-item active">
```

Comment:
at a time only 1 link can be active, we will have to write logic for adding that based on url path.